.github/workflows/intel.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         export CC=$(which icx)
         export CXXFLAGS="-fsycl ${CXXFLAGS}"
 
-        cmake -S . -B build_sp         \
+        cmake -S . -B build_dpsp       \
```

Comment:
Actually, could you change also the names of the job, namely both `build_dpcc` and the `name` attribute in
```
  build_dpcc:
    name: oneAPI DPC++ SP
```
or at least the name attribute, which is the one that gets displayed?

A suggestion could be `build_dpc_dpsp` or simply `build_dpc` and `name: oneAPI DPC++ DP SP`.